pkg/controller/core/workload_controller_test.go

```diff
@@ -590,12 +606,6 @@ func TestReconcile(t *testing.T) {
 				}).
 				RequeueState(ptr.To[int32](1), ptr.To(metav1.NewTime(testStartTime.Add(1*time.Second).Truncate(time.Second)))).
 				Obj(),
-			wantEvents: []utiltesting.EventRecord{{
-				Key:       types.NamespacedName{Name: "wl", Namespace: "ns"},
-				EventType: v1.EventTypeNormal,
-				Reason:    kueue.WorkloadEvictedByDeactivation,
-				Message:   "Deactivated Workload \"ns/wl\" by reached re-queue backoffLimitCount",
-			}},
```

Comment:
What is the reason that the workload doesn't have this event?
This workload should have the "EvictedDueToInactiveWorkload", right?

Reply:
Yes, we moved it to https://github.com/kubernetes-sigs/kueue/blob/4a5c0612ff80dc0595364dcfadddb1b0e8a85637/pkg/controller/core/workload_controller.go#L210-L211. So on this test case we can't get this event.

Reply:
Sorry? I couldn't understand the reason why this workload doesn't have the event something like:

```YAML
{
	Key:       types.NamespacedName{Name: "wl", Namespace: "ns"},
	EventType: corev1.EventTypeNormal,
	Reason:    "EvictedDueToInactiveWorkload",
	Message:   "The workload is deactivated",
},
```

Reply:
NVM
https://github.com/kubernetes-sigs/kueue/pull/2376#discussion_r1634990476

Reply:
This workload should have event on the next reconcile.